### PR TITLE
OFFAPPS-453 set fields in the right order, make user fields editable conditionally

### PR DESCRIPTION
**Pull request by sandlerr**
:koala:

Third time's a charm

/cc @zendesk/quokka

### References
 - Jira link: https://zendesk.atlassian.net/browse/OFFAPPS-453

### Risks
 - None

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -245,14 +244,13 @@
...
 
-    setOrgEditable: function() {
-      if (this.orgEditable !== undefined) {
-        return;
-      }
+    setEditable: function() {
       var role = this.currentUser().role();
-      this.orgEditable = { general: false, notes: false };
+      this.orgEditable = { general: false, notes: true };
+      this.userEditable = false;
```
Comment (sandlerr) [inline app.js]: previously user fields always showed as editable but this is quite locked down based on your role

File: app.js [hunk 2]
```diff
@@ -218,8 +218,7 @@
         orgFields: this.fieldsForCurrentOrg(),
         orgFieldsActivated: this.storage.user && this.storage.orgFieldsActivated && this.storage.user.organization,
         org: this.storage.user && this.storage.user.organization,
-        orgTickets: this.makeTicketsLinks(this.storage.orgTicketsCounters),
-        orgEditable: this.orgEditable
```
Comment (sandlerr) [inline app.js]: unused variable in template

File: app.js [hunk 3]
```diff
@@ -273,14 +271,14 @@
       };
       this.storage = _.clone(defaultStorage); // not sure the clone is needed here
       this.storage.orgFieldsActivated = (this.setting('orgFieldsActivated') == 'true');
-      var defaultSelection = '["##builtin_tags", "##builtin_notes", "##builtin_details"]';
+      var defaultSelection = '["##builtin_tags", "##builtin_details", "##builtin_notes"]';
```
Comment (sandlerr) [inline app.js]: re-arrange as per organisations page from zendesk_crm I think

File: app.js [hunk 4]
```diff
@@ -528,20 +524,20 @@
...
-          editable: this.orgEditable.notes
+          editable: this.orgEditable.general
         },
         {
-          key: '##builtin_details',
-          title: this.I18n.t('details'),
+          key: '##builtin_notes',
+          title: this.I18n.t('notes'),
           description: '',
-          position: Number.MAXVALUE,
```
Comment (sandlerr) [inline app.js]: fun fact: this was NaN

**Comment by adammw** [comment]
:+1:

**Comment by sadia** [comment]
:+1:

**Pull request merged**